lib/math/math.go

```diff
@@ -0,0 +1,100 @@
+package math
```

Comment:
Package needs doc comment. Follow the example of the json package for what form this should take.